django/db/models/functions/datetime.py

```diff
@@ -207,13 +207,9 @@ def resolve_expression(self, query=None, allow_joins=True, reuse=None, summarize
     def convert_value(self, value, expression, connection):
         if isinstance(self.output_field, DateTimeField):
             if settings.USE_TZ:
-                if value is None:
-                    raise ValueError(
```

Comment:
This wasn't tested according to the coverage, and I can't see why we wouldn't allow `None` to be returned...